assets/src/scripts/monitoring-location/components/hydrograph/download-links.test.js

```diff
@@ -39,33 +39,28 @@ xdescribe('monitoring-location/components/hydrograph/download-links', () => {
 
         it('creates an unordered list and the correct number of list items and hyperlinks when only current time series is showing', () => {
             const TEST_STATE = {
-                'ivTimeSeriesData': {
-                    'queryInfo': {
-                        'current:P7D': {
-                            'queryURL': 'http://waterservices.usgs.gov/nwis/iv/sites=05370000&period=P7D&siteStatus=all&format=json'
-                        }
-                    }, 'variables': {
-                        '45807042': {
-                            'variableCode': {
-                                'value': '00060'
-                            }
-                        },
-                        '450807142': {
-                            'variableCode': {
-                                'value': '00010'
-                            }
-                        }
-                    }
+                'hydrographData': {
```

Comment:
You might consider creating a single TEST_STATE using some of the constants in mock-hydrograph-state and then use the spread operator to customize for each test. This is the pattern I used for most of the tests in components/hydrograph.

Reply:
I used a test state for some and cleaned out some unneeded items of for others. Hopefully, it as lean as it can be.